assets/js/modules/ads/components/settings/SettingsEdit.stories.js

```diff
@@ -172,3 +178,38 @@ IcePaxEnabled.decorators = [
 		);
 	},
 ];
+
+export const FirstPartyModeEnabled = Template.bind( null );
+FirstPartyModeEnabled.storyName = 'FirstPartyModeEnabled';
+FirstPartyModeEnabled.decorators = [
+	( Story ) => {
+		const setupRegistry = ( registry ) => {
+			const fpmServerRequirementsEndpoint = new RegExp(
+				/^\/google-site-kit\/v1\/core\/site\/data\/fpm-server-requirement-status/
```

Comment:
@ankitrox, please use the following format for consistency.
```suggestion
				'^/google-site-kit/v1/core/site/data/fpm-server-requirement-status'
```